Formula/gocket.rb

```diff
@@ -0,0 +1,28 @@
+class Gocket < Formula
+  desc "CLI / TUI for Pocket"
+  homepage "https://github.com/Phantas0s/gocket"
+  url "https://github.com/Phantas0s/gocket/releases/download/v0.2.4/gocket_0.2.4_Darwin_x86_64.tar.gz"
```

Comment:
https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

Reply:
Fair point, I changed it to build from source.